jumpstart-foundation-models/text2text-generation-flan-t5-xl/text2text-generation-flan-t5-xl.ipynb

```diff
@@ -0,0 +1,1455 @@
+{
```

Comment:
<p>Let's split it in two:  1/ common sense reasoning/ natural language inference, 2/ Question-answering which you have demonstrated below with a different prompt.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3795/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>